test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -156,10 +169,7 @@ public void execute_add_successful() throws Exception {
 
         // execute command and verify result
         assertCommandBehavior(helper.generateAddCommand(toBeAdded),
-                              String.format(AddCommand.MESSAGE_SUCCESS, toBeAdded),
```

Comment:
I'm iffy on the changes you're making to the parameter style here. For large number of parameters it's actually good to list them this way. Ctrl+F "stacking" https://oss-generic.github.io/process/codingstandards/coding-standards-java.html